### PR TITLE
hashtable unit test and code complete

### PR DESCRIPTION
**Pull request by jennisung**

### CODE REVIEW

**Comment by jennisung** [comment]
- [na ] Top-level README “Table of Contents” is updated
 - [ ] README for this challenge is complete
       - [ ] Summary, Description, Approach & Efficiency, Solution
       - [ ] Picture of whiteboard
       - [ ] Link to code
 - [x ] Feature tasks for this challenge are completed
 - [x ] Unit tests written and passing
       - [x ] “Happy Path” - Expected outcome
       - [ x] Expected failure
       - [ x] Edge Case (if applicable/obvious)

File: java/datastructures/lib/src/test/java/datastructures/hashtable/hashMapTest.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    hashMap.set("keyTwo", 2);
+
+    assertEquals(1, hashMap.get("keyOne"));
+    assertEquals(2, hashMap.get("keyTwo"));
+  }
+
+    @Test
+    public void doubleKeysTest() {
+      hashMap.set("keyOne", 1);
+      hashMap.set("keyTwo", 2);
```
Comment (lucy-gelderloos) [inline java/datastructures/lib/src/test/java/datastructures/hashtable/hashMapTest.java:30]: I think you mean `set("keyOne", 2)` here; with that correction, the test passes.
